### PR TITLE
[application] SimpleShot application update for V2

### PR DESCRIPTION
**Pull request by kparichay**
SimpleShot application update for layer V2 design.
This includes updating the three implemented layers - centering, l2norm, and centroid_knn.
Further, task_runner has been updated to use only 1 loss kind of layer.

Unittests of layers have not been enabled, they will be enabled with unittest_layers.

Signed-off-by: Parichay Kapoor <kparichay@gmail.com>

### CODE REVIEW

File: Applications/SimpleShot/layers/centering.cpp [hunk 1]
```diff
@@ -69,15 +62,18 @@ void CenteringLayer::read(std::ifstream &file) {
...
-void CenteringLayer::forwarding(bool training) {
-  std::cout << net_input[0]->getVariableRef().getDim();
-  std::cout << net_hidden[0]->getVariableRef().getDim();
+void CenteringLayer::forwarding(nntrainer::RunLayerContext &context,
+                                bool training) {
+  auto &hidden_ = context.getOutput(SINGLE_INOUT_IDX);
+  auto &input_ = context.getInput(SINGLE_INOUT_IDX);
+
+  std::cout << input_.getDim();
+  std::cout << hidden_.getDim();
```
Comment (jijoongmoon) [inline Applications/SimpleShot/layers/centering.cpp:71]: It is just for the debugging?

**Comment by jijoongmoon** [comment]
I think ```Application/KNN/``` also need to update with KNN Layer as well. #1421

**Review by jijoongmoon — APPROVED** [review]
LGTM

File: Applications/SimpleShot/layers/centering.cpp (hunk 1 above)
Comment (kparichay) [reply Applications/SimpleShot/layers/centering.cpp:71]: this existed in the original code, but removed now.

**Comment by kparichay** [comment]
> I think `Application/KNN/` also need to update with KNN Layer as well. #1421

Right, that should also be done. How about in main branch? Maybe this can be done after KNN layer is moved to nntrainer core?

**Review by zhoonit — APPROVED** [review]
LGTM

**Pull request merged**